pages/event-types/[type].tsx

```diff
@@ -49,6 +49,9 @@ import classNames from "@lib/classNames";
 import { inferSSRProps } from "@lib/types/inferSSRProps";
 import { asStringOrThrow } from "@lib/asStringOrNull";
 import Button from "@components/ui/Button";
+import { useTranslation } from "next-i18next";
```

Comment:
i wonder, do we wanna abstract all of these imports or just continue directly importing?